docs/examples.rst

```diff
@@ -0,0 +1,179 @@
+    >>> SAMPL_tasks, SAMPL_datasets, transformers = dc.molnet.load_sampl()
+    >>> SAMPL_tasks
+    ['expt']
+    >>> train_dataset, valid_dataset, test_dataset = SAMPL_datasets
+    >>>
+    >>> # We want to know the pearson R squared score, averaged across tasks
+    >>> avg_pearson_r2 = dc.metrics.Metric(dc.metrics.pearson_r2_score, np.mean)
+    >>> 
+    >>> # We'll train a multitask regressor (fully connected network)
+    >>> model = dc.models.MultitaskRegressor(
+    ...     len(SAMPL_tasks),
+    ...     n_features = 1024,
+    ...     layer_sizes=[1000],
+    ...     dropouts=[.25],
+    ...     learning_rate=0.001,
+    ...     batch_size=50)
+    >>> 
+    >>> # Fit trained model (returns average loss over the most recent checkpoint interval)
+    >>> model.fit(train_dataset)
+    0.1726440668106079
+    >>> 
+    >>> # We now evaluate our fitted model on our training and validation sets
+    >>> model.evaluate(train_dataset, [avg_pearson_r2], transformers)
+    {'mean-pearson_r2_score': 0.9244964295814636}
+    >>> model.evaluate(valid_dataset, [avg_pearson_r2], transformers)
+    {'mean-pearson_r2_score': 0.7532658569385681}
+
+
+GraphConvModel
+^^^^^^^^^^^^^^
+The default `featurizer <./featurizers.html>`_ for SAMPL is :code:`ECFP`, short for
+`"Extended-connectivity fingerprints." <./featurizers.html#circularfingerprint>`_
+For a :class:`GraphConvModel <deepchem.models.GraphConvModel>`, we'll reload our datasets with :code:`featurizer='GraphConv'`:
+
+.. doctest:: sampl
+
+    >>> seed_all()
+    >>> # Load SAMPL dataset
+    >>> SAMPL_tasks, SAMPL_datasets, transformers = dc.molnet.load_sampl(
+    ...     featurizer='GraphConv')
+    >>> train_dataset, valid_dataset, test_dataset = SAMPL_datasets
+    >>>
+    >>> model = dc.models.GraphConvModel(len(SAMPL_tasks), mode='regression')
+    >>> 
+    >>> # Fit trained model (returns average loss over the most recent checkpoint interval)
+    >>> model.fit(train_dataset, nb_epoch=20)
+    0.05753047466278076
+    >>> 
+    >>> # We now evaluate our fitted model on our training and validation sets
+    >>> model.evaluate(train_dataset, [avg_pearson_r2], transformers)
+    {'mean-pearson_r2_score': 0.5772751202910659}
+    >>> model.evaluate(valid_dataset, [avg_pearson_r2], transformers)
+    {'mean-pearson_r2_score': 0.36771456280565507}
+
+
+..
+  ChEMBL
+  -------
+
+  Examples of training models on `ChEMBL <https://www.ebi.ac.uk/chembl/>` dataset included in `MoleculeNet <./moleculenet.html>`_.
+
+    ChEMBL is a manually curated database of bioactive molecules with drug-like properties.
+    It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs.
+
+  MultitaskRegressor
+  ^^^^^^^^^^^^^^^^^^
+
+  .. doctest:: chembl
+
+      >>> seed_all()
+      >>> # Load ChEMBL 5thresh dataset with random splitting
+      >>> chembl_tasks, datasets, transformers = dc.molnet.load_chembl(
```

Comment:
Ok this is good to know we have a failure here! I'll give this a try locally in a bit and see if I'm able to reproduce the training failure